crates/forge/src/running.rs

```diff
@@ -188,17 +188,22 @@ pub(crate) fn run_test_case(
             &case.fork_config,
         )?,
     };
-    let mut context = build_context();
+    let block_info = state_reader.get_block_info().unwrap();
```

Comment:
It should result in a failed test, not panic the runner. The best option would be make `run_test_case` return the result of `run_function` and match the error in `blocking_run_from_test`. This way we can `map_err` here and bubble the error